client/extensions/woocommerce/app/shipping/index.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Probably want to put all this in `app/settings/shipping` to mirror what is now in place for checkout/payment settings ( see https://github.com/Automattic/wp-calypso/pull/13866/files#diff-0ce89feda2e69a9bf3df1371d3d7573dR1 )